lib/document-downloader.js

```diff
@@ -38,6 +39,8 @@ DocumentDownloader.install = function (dest, content) {
 DocumentDownloader.installAll = function (destsContents) {
   // `e` is a tuple of [dest, content]
   return Promise.all(destsContents.toArray().map(function (e) {
+    var path = require('path').dirname(e[0]);
+    Mkdirp.sync(path);
```

Comment:
This sync version is fine by me.
I don't think it takes a lot longer to execute but I'd be curious to see the difference. Meanwhile it makes the code easier to read than a callback-based async version. When (if!) `mkdirp` [supports Promises](https://github.com/substack/node-mkdirp/issues/86), we can probably do something like:

```js
return Mkdirp(path).then(function ()
  return DocumentDownloader.install(e[0], e[1]);
});
```